dashboard/app/helpers/levels_helper.rb

```diff
@@ -261,8 +261,10 @@ def app_options
     view_options(backpack_enabled: backpack_enabled)
 
     if backpack_enabled
+      # Backpack is used in lab2 apps also but app_options is only used by legacy labs, i.e., Javalab.
+      game_id = Game.by_name('Javalab')
```

Comment:
we could use `@level.game` instead here to be more future-proof

Reply:
Thanks! I think I'll go with ` @level.game_id`.